### PR TITLE
Task/54214 objc demo app

### PR DESCRIPTION
**Pull request by asenthil-altmtrk**

### CODE REVIEW

File: Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~HEAD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+/// Representation of the PayPal account query parameters.
+public class HyperwalletPayPalAccountQueryParam: HyperwalletTransferMethodQueryParam {
```
Comment (gmeyer-hw) [inline Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~HEAD]: You can remove this file

File: Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~origin_master [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+/// Representation of the PayPal account query parameters.
+public class HyperwalletPayPalAccountQueryParam: HyperwalletTransferMethodQueryParam {
```
Comment (gmeyer-hw) [inline Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~origin_master]: You can remove this file

File: Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~HEAD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+/// Representation of the prepaid card query parameters.
+public class HyperwalletPrepaidCardQueryParm: HyperwalletTransferMethodQueryParam {
```
Comment (gmeyer-hw) [inline Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~HEAD]: You can remove this file

File: Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~origin_master [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+/// Representation of the prepaid card query parameters.
+public class HyperwalletPrepaidCardQueryParm: HyperwalletTransferMethodQueryParam {
```
Comment (gmeyer-hw) [inline Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~origin_master]: You can remove this file

File: Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~HEAD (hunk 3 above)
Comment (asenthil-altmtrk) [reply Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~HEAD]: Removed

File: Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~origin_master (hunk 2 above)
Comment (asenthil-altmtrk) [reply Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~origin_master]: Removed

File: Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~HEAD (hunk 1 above)
Comment (asenthil-altmtrk) [reply Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift~HEAD]: Removed

File: Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~origin_master (hunk 4 above)
Comment (asenthil-altmtrk) [reply Sources/Model/TransferMethod/HyperwalletPrepaidCardQueryParam.swift~origin_master]: removed

File: Sources/HyperwalletAuthenticationTokenProvider.swift [hunk 5]
```diff
@@ -38,7 +38,7 @@ public protocol HyperwalletAuthenticationTokenProvider {
     ///   - authenticationToken: a JWT token identifying a Hyperwallet User principal
     ///   - error: an `HyperwalletAuthenticationErrorType` indicating the cause of the authentication
```
Comment (jsaini-hw) [inline Sources/HyperwalletAuthenticationTokenProvider.swift]: Update the comment as well

**Comment by asenthil-altmtrk** [comment]
Updated the comments

**Pull request closed without merging**